covid-frontend/.env

```diff
@@ -1,2 +0,0 @@
-REACT_APP_ENV=development
```

Comment:
@sfakir can we get rid of these here or what is are they used for?

Reply:
@tholor  The `.env` file is ignored in the [.gitignore](https://github.com/deepset-ai/COVID-QA/blob/b0588f38b051b7ab89cedb3ff184878f8674538d/covid-frontend/.gitignore#L11) but wasn't removed from tracking. That's why I removed it.

Reply:
yes, the .env file can be removed, I think it's obvious but I will add to the readme that a dev has to copy a sample .env to connect to the backend (local or staging)